### DISCUSSION TITLE
Permission to use Open States icon when linking?

### DISCUSSION THREAD

**Discussion by earthboundkid** [openstates/issues#660] [Q&A] [answered]
Is it acceptable to use the Open States SVG icon when linking to the site, a la a Facebook f or Twitter bird?

**Comment by adamprice2** [comment]
Assuming you mean the icon found here:-
https://github.com/openstates/openstates.org/blob/develop/static/images/openstates_logo_icon.svg

Then it is licensed under a MIT license. (https://github.com/openstates/openstates.org/blob/develop/LICENSE)

So it seems likely you can use it in almost any way you want.

No harm in asking though if you are concerned.

**Reply by earthboundkid** [reply]
Often for open source projects, there will be a trademark separate from a license, so for example, anyone can build Firefox, but if you distribute it you need to use a different icon/name. I think using the icon would probably just fall under fair use anyway, but figured it couldn't hurt to ask.

**Comment by NewAgeAirbender** [comment, marked as answer]
Hey, yeah that icon is okay to use!

**Reply by earthboundkid** [reply]
Thank you!
